referenceframe/transformable.go

```diff
@@ -98,8 +98,8 @@ func (lF *LinkInFrame) Geometry() spatialmath.Geometry {
 	return lF.geometry
 }
 
-// ToStaticFrame converts a LinkInFrame into a staticFrame with a new name.
-func (lF *LinkInFrame) ToStaticFrame(name string) (Frame, error) {
+// toStaticFrame converts a LinkInFrame into a staticFrame with a new name.
+func (lF *LinkInFrame) toStaticFrame(name string) (Frame, error) {
```

Comment:
Why is this becoming private?
The function name isn't great but really it's a function to deep copy a LinkInFrame with a new name. Since LinkInFrame is public this seems reasonable to have. Would be fine with naming it something more descriptive.

Reply:
I understand that it isn't used elsewhere in RDK; but how do we know that it isn't being used elsewhere?

> This was part of the changes I made earlier and isn't needed anymore.

This particular function has been public and usable for two years.